six/modules/c++/six/include/six/WriteControl.h

```diff
@@ -175,10 +176,9 @@ class WriteControl
     }
 
     /*!
-     *  const pointer to Container.  This object is not owned by the
-     *  WriteControl
+     *  shared const pointer to Container.
      */
-    const Container* getContainer()
+    mem::SharedPtr<const Container> getContainer()
```

Comment:
Since you have a non-const version in `ReadControl`, might as well put that version in too